examples/serverless-vanilla/unit-test/handler.test.js

```diff
@@ -0,0 +1,75 @@
+const handler = require("../handler").handler;
+const comparisonHandler = require("../comparison_handler").handler;
+
+describe("handler", () => {
```

Comment:
Normally what I encourage developers to do, especially when Laconia.js is used, is to unit-test from `app` level. One of the example can be seen here: https://github.com/laconiajs/laconia/blob/master/packages/laconia-acceptance-test/test/capture-card-payment.spec.js

This help shows the comparison when Laconia.js is used as dependency injection is encouraged, and it's quite hard to mock dependencies from the handler level, especially when there are other interaction to cloud services like S3 or DynamoDB.

If there's a need to mock from handler level, with Laconia.js, you can override the dependencies by calling `register` too i.e. `handler.register(overrideInstances)`. Although this would mean that the factories are still processed, and if there's an externall call made the unit test will still fail.

Reply:
Ah right, this makes sense. Yeah because I'm basically unit testing a third party library. I will update this.